src/makeStyles.tsx

```diff
@@ -23,7 +23,7 @@ export function createMakeStyles<Theme>(params: { useTheme: () => Theme }) {
 
     /** returns useStyle. */
     function makeStyles<
-        Params = void,
+        ParamsType = void,
```

Comment:
I may prefer make ParamsType default value equals to any to make it optional to inject Params type, but you may want to keep it void fo have safer as you said in the previous PR.